src/modules/auth-module/components/register-form.tsx

```diff
@@ -45,8 +51,18 @@ function RegisterForm({ onSuccessfullSignUp }: IProps) {
         async (data: FormData) => {
             if (isValid) {
                 try {
-                    await signUp(data.email, data.password);
-                    onSuccessfullSignUp();
+                    const user = await signUp(data.email, data.password);
+                    await createWithDocRef(user.user.uid, {
+                        birthdate: Firestore.Timestamp.now(),
+                        firstName: '',
+                        gender: '',
+                        interests: [],
+                        lastName: '',
+                        profileImageRef: '',
+                        userId: user.user.uid,
```

Comment:
we dont need the userId as a specific field inside user document because we can jusst retrieve the uid with auth.currentUser.uid

Reply:
We had that inside our model, so i just followed that.

Reply:
yeah i made that one optional now cause for creating a new user in db this wont be required